packages/solid/src/validation/url.ts

```diff
@@ -12,9 +12,9 @@ type Value = MaybeValue<string>;
 export function url(error: string): (value: Value) => string {
   return (value: Value) => {
     try {
-      value && new URL(value);
+      new URL(value);
       return '';
-    } catch (_) {
+    } catch {
```

Comment:
The `exceptionvar` [is optional ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch#exceptionvar)

Reply:
Good catch!